Gruntfile.js

```diff
@@ -99,7 +99,11 @@ module.exports = function (grunt) {
     grunt.registerTask('compile:js', function (fullCompile) {
         grunt.task.run(['clean:js', 'compile:inlineSvgs']);
 
-        grunt.task.run(['concurrent:requireJS', 'copy:javascript']);
+        if (!options.isDev) {
```

Comment:
i think if a dev is running this task, then it should do the `concurrent:requireJS` task. Why do we need the guard? is it for a watch?